src/pscoast.c

```diff
@@ -463,6 +473,11 @@ GMT_LOCAL int parse (struct GMT_CTRL *GMT, struct PSCOAST_CTRL *Ctrl, struct GMT
 				break;
 			case 'W':
 				Ctrl->W.active = true;	/* Want to draw shorelines */
+				if (strchr (opt->arg, '/') == NULL && strchr (opt->arg, ',')) {
```

Comment:
Some tests fail due to a bug introduced here. The syntax of -W option is **-W**[_level_/]_pen_.

Reply:
Correct, too much copy/paste on this one.